src/_grid.scss

```diff
@@ -0,0 +1,93 @@
+/* List of columns in word and number.
+ * These numbers will be looped through in  the mixins to create the class name and 
+ * the magnitude of columns the column will span.
+ */
+$divisions: ('one' 1, 'two' 2 , 'three' 3, 'four' 4, 'five' 5, 'six' 6, 'seven' 7, 'eight' 8, 'nine' 9, 'ten' 10, 'eleven' 11, 'twelve' 12);
+
+/* 12 column grid - default gutter 1em */
+.grid {
+    display: grid;
+    grid-template-columns: repeat(12, [col] 1fr);
+    grid-auto-rows: minmax(50px, auto);
+    grid-gap: 1em;
+}
+
+/* No gutter helper class */
+.gutter-none {
```

Comment:
Gutter should be specific to `.grid` only, as it exceeds the `vw` if used with all columns.

Reply:
@piccolazucca another great spot. Fixed that now too.